ai-ml/gke-ray/raytrain/pytorch-mnist/ray-cluster.yaml

```diff
@@ -52,12 +52,12 @@ spec:
       spec:
         containers:
         - name: ray-worker
-          image: rayproject/ray:2.9.0
+          image: rayproject/ray:2.37.0
           resources:
             limits:
-              cpu: 2
+              cpu: 3
```

Comment:
Can we use 4 instead?

Reply:
Tested it with 4 and it works, added that change in 349da28.